src/mc-fle2-payload-iev-private-v2.h

```diff
@@ -68,14 +68,14 @@
  *   uint8_t fle_blob_subtype = 15;
  *   uint8_t S_KeyId[16];
  *   uint8_t original_bson_type;
- *   uint8_t edge_count;
+ *   uint32_t edge_count;
```

Comment:
Since this seems to be referencing the structure of the payload, rather than how we are storing it, I think this should stay uint8_t.